atomicapp/utils.py

```diff
@@ -197,6 +197,36 @@ def sanitizePath(path):
             return path[7:]
 
     @staticmethod
+    def run_cmd(cmd, args_str=None):
```

Comment:
add arg that allows for specifying if we should check the exit code or not. ignore_exit_code=False or something like that.